azerothcore/azerothcore-wotlk
# fix(Core/Spells): Remove vanish immunity aura on stealth removal.

Fixes #2668



## Issues Addressed:

- Closes #2668

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

Create 3 macros:
1. `.cast 1856`
2. `/cancelaura Stealth`
3. `.cast 16566`
Use all these macros one by one.
At the end, the root should be applied on you.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
